libusb/libusbi.h

```diff
@@ -1021,6 +1023,29 @@ struct usbi_os_backend {
 	int (*get_device_list)(struct libusb_context *ctx,
 		struct discovered_devs **discdevs);
 
+	/* Retrieve a device string without needing to open the device.
+	 *
+	 * The string should be retrieved without opening the device
+	 * and ideally without performing USB transactions to the device.
+	 * Most operating systems read and cache the common string 
+	 * descriptors.  Use the OS-specific calls to retrieve these strings.
+	 *
+	 * Since the USB string descriptor could be processed by the OS,
+	 * this function returns a UTF-8 encoded string.  The string will
+	 * be returned untranslated or in the default OS language
+	 * when supported by the OS and USB device.
```

Comment:
This part confused me... At first I thought "untranslated" referred to a translation from the OS encoding (ex: UTF-16) to UTF-8.  But now I think you're talking about human languages?  If so, maybe start a new paragraph.

Reply:
Yes, human language.  USB supports multiple languages for string descriptors, but the OS will only share one version through this API.  We are at the mercy of the OS to share whatever string it thinks is right.

Will move to separate paragraph.